.bones/tasks/marky-waw.md

```
@@ -0,0 +1,403 @@
+id: marky-waw
+title: 'Task 2: Wire block-refs and journal detection into DocumentIndex'
+status: closed
+type: task
+priority: 1
+owner: sethyanow@users.noreply.github.com
+depends_on: [marky-khy]
+parent: marky-9mo
+---
+
+
+
```

markdownlint-cli2 (0.21.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)